zebra/zebra_rib.c

```diff
@@ -1136,6 +1136,13 @@ static void rib_process(struct route_node *rn)
 							 ROUTE_ENTRY_REMOVED);
 				}
 
+				const struct prefix *p;
```

Comment:
don't we usually declare locals at the start of a function - is this a new thing?